### PR TITLE
Warn user when projectdir is used and no project is active

### LINKED ISSUE #10: Give warning on `projectdir` if project is standard Julia project
if there is a way to consistently tell if the current project is the "julia default" (whatever that means) then we should warn when the function `projectdir()` is called and showing the default dir.

### PR DESCRIPTION
**Pull request by tamasgal**
This is just a straightforward implementation using the hints in #10.

Closes #10

### CODE REVIEW

**Comment by Datseris** [comment]
Cool, thanks! 

Could you please:
1. add some basic tests
2. Increment minor version in `Project.toml`
3. Add this new function to the changelog

File: src/project_setup.jl [hunk 1]
```diff
@@ -7,6 +7,16 @@ export findproject, quickactivate
 
 @deprecate scriptdir scriptsdir #TODO: remove in next release
 
+"""
+    function is_standard_julia_project()
+
+Returns true if a project other than the standard Julia project is active.
```
Comment (Datseris) [inline src/project_setup.jl]: I think it returns `true` if the standarrd Julia project is active, not other than the standard.

Comment (tamasgal) [reply src/project_setup.jl]: Yeah sure! It was a bit late 😉

**Comment by tamasgal** [comment]
Ahm, Travis is calling the test suite with `--project=...`? My dead simple test functions will then of course fail ;)

**Comment by Datseris** [comment]
Notice that you can simply call `Pkg.activate()` and this will activate the default Julia project I think. Maybe it is worth trying? I am not _really_ sure what this will do on travis...

**Comment by tamasgal** [comment]
Yeah let's see, it works locally at least ;)

**Comment by Datseris** [comment]
unfortunately it didnt work :(

**Comment by tamasgal** [comment]
Yeah I noticed but I had no time yet to look at it. It's a pity that it works locally... I'll look at it later, today I really need to write a few pages in my PhD thesis 🙈

**Comment by asinghvi17** [comment]
it's not v#.#, that was just a placeholder :)

**Comment by tamasgal** [comment]
Oh, I was already wondering what kind of string magic that is 😅

**Comment by asinghvi17** [comment]
Well, here's the problem:
> `Base.active_project() = "/tmp/tmpOvxF4V/Project.toml"`

so the test subprocess can't seem to affect the standard project.  I wonder if you could manually activate the standard project using the load_path_expand trick?

**Comment by Datseris** [comment]
Maybe travis doesn't have a "standard project"?

**Comment by asinghvi17** [comment]
Possible but unlikely - try running that in a normal Julia command.

**Comment by Datseris** [comment]
For me it is 
```

julia> Base.active_project()
"C:\\Users\\datseris\\.julia\\environments\\v1.1\\Project.toml"
```

**Comment by asinghvi17** [comment]
I meant in Travis, sorry for the confusion :)

**Comment by sebastianpech** [comment]
I think I read something about that test basically run in a sandboxed mode. Expanding the load path with `@v#.#` should be fine, it's also what you find in the julia `LOAD_PATH` variable.
```
julia> LOAD_PATH
4-element Array{String,1}:
 "@"
 "@v#.#"
 "@stdlib"
 "./modules/"
```

**Comment by asinghvi17** [comment]
Cool, I guess that will have to remain untested.  It _is_ only a warning...

**Comment by asinghvi17** [comment]
We could also test with `withenv`.

**Comment by sebastianpech** [comment]
Isn't `withenv` just used for temporary changes in `ENV`?

**Comment by sebastianpech** [comment]
We could leave the test that shouldn't generate a warning. This way we at least get an error if some of the julia path expansion internals are changed.

**Comment by Datseris** [comment]
Sure, I agree with this as well.

**Comment by Datseris** [comment]
So should we revert the change to `@v#.#` ?

**Comment by sebastianpech** [comment]
Yes. For details you can have a look at [Fredrik's julia con talk](https://youtu.be/q-LV4zoxc-E?list=PLP8iPy9hna6StY9tIJIUN3F_co9A0zh0H&t=490). He explains the expansion of `LOAD_PATH` there.

**Comment by Datseris** [comment]
Urgh, sorry I messed up through GitKraken. Anyways I think current master has all the updates :P

**Comment by fredrikekre** [comment]
The Package itself is the active project on CI.

**Comment by Datseris** [comment]
Thanks for the info!

**Pull request merged**